src/main/java/com/connexta/ddf/catalog/direct/CatalogMethods.java

```diff
@@ -98,21 +99,25 @@
   //  private final transient SimpleDateFormat dateFormat = new
   // SimpleDateFormat(ISO_8601_DATE_FORMAT);
 
-  private final Map<String, DocMethod> METHODS;
+  private final SubjectOperations subjectOperations;
+
+  private final RpcFactory rpc = new RpcFactoryImpl();
```

Comment:
The RpcFactory is designed to be use as a service in external bundles. Since this bundle exports it there is not a need to export the service and then reimport it.